src/styles/components/header.scss

```diff
@@ -1,16 +1,16 @@
 .header {
-  flex: 0 1;
-  display: flex;
-  background-color: $mainDark-color;
   position: relative;
-  justify-content: flex-end;
+  display: flex;
+  flex: 0 1;
   align-items: center;
+  justify-content: flex-end;
   padding: 20px $defaultSidePadding;
+  background-color: $mainDark-color;
```

Comment:
One name convention pattern:
$default-side-padding
$main-dark-color
or
$defaultSidePadding
$mainDarkColor

PS: If I needed to give name to color or something else in scss I would choose dashed style 'your-variable-name'